### PR TITLE
feat(issue-commands): add command to assign issues to projects automatically

### PR DESCRIPTION
**Pull request by tolzhabayev**
Fixes https://github.com/grafana/grafana-github-actions/issues/29

**Why?**
https://github.com/grafana/grafana-github-actions/issues/29

PR introduces new action to the type `label` called `addToProject` which automates assigning issues and pull requests to projects based on a label

**Why can't we assign "columns" automatically?**
New types of github projects do not have pre-defined columns or statuses. It is just another custom field, which would mean you would need to get a lot of custom information like `node id` for the status column and `node id` for the status value and only then you might be able to properly assign it. Further, the new projects are in beta and soon they will have more custom automation inside which would make it more proper to use the "inside project automation" instead of the grafana bot

**Additional changes**
- fixing linting errors

**Breaking changes**
- none

**How to battle-test it?**
**Issues**: Go to [this issue ](https://github.com/grafana/github-actions-testrepo/issues/31) and add a label `plugins`, wait a bit and it should be added to [this project](https://github.com/orgs/grafana/projects/76/views/12) automatically
**PRs**: Go to [this PR ](https://github.com/grafana/github-actions-testrepo/pull/32) and add a label `plugins`, wait a bit and it should be added to [this project](https://github.com/orgs/grafana/projects/76/views/12) automatically

### CODE REVIEW

File: update-changelog/index.ts [hunk 1]
```diff
@@ -44,7 +44,18 @@ class UpdateChangelog extends Action {
 		fileUpdater.writeFile(changelogFile)
 
 		await writeDocsFiles({ version, builder })
-		await npx('prettier', '--no-config', '--trailing-comma', 'es5', '--single-quote', '--print-width', '120', '--list-different', '**/*.md', '--write')
+		await npx(
```
Comment (tolzhabayev) [inline update-changelog/index.ts:47]: Note: was done by linter...

File: tsconfig.test.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
 		]
 	},
 	"include": [
-		"**/*.test.ts"
+		"**/*.test.ts",
+		"__mocks__/**/*.ts"
```
Comment (tolzhabayev) [inline tsconfig.test.json:14]: Note: was needed to silence the linter about `@actions/github.ts` file in `__mocks__` folder

File: package.json [hunk 3]
```diff
@@ -56,7 +57,7 @@
 	},
 	"husky": {
 		"hooks": {
-			"pre-commit": "npm run build && git add \"**/*.js\""
+			"pre-commit": "npm run lint && npm run build && git add \"**/*.js\""
```
Comment (tolzhabayev) [inline package.json:60]: Note: I think adding linting would help as we don't have any CI/CD that runs the checks

File: api/octokit.ts [hunk 4]
```diff
@@ -6,19 +6,24 @@
...
 
 export class OctoKit implements GitHub {
 	private _octokit: GitHubAPI
+	private _octokitGraphQL: typeof graphql
 	public get octokit(): GitHubAPI {
 		numRequests++
 		return this._octokit
 	}
-
+	public get octokitGraphQL() {
```
Comment (tolzhabayev) [inline api/octokit.ts:24]: Note: not really necessary, I know. But will be helpful for future actions.

**Review by marefr — COMMENTED** [review]
Quick comment, I'm going to look into more details next week. It works as expected :tada: And a quick glance I can see a lot of valuable improvements which is great :+1: 

Things to consider:
- We already have the type `label` so maybe should consider moving type `addToProject` to the `action` instead. This means that existing types basically could use the action `addToProject` as well, for example .
- Does it support both org and repo projects?

**Comment by tolzhabayev** [comment]
> * We already have the type `label` so maybe should consider moving type `addToProject` to the `action` instead. This means that existing types basically could use the action `addToProject` as well, for example .
I will see if I can fit in in there. It is currently a bit chaotic what type `label` really does and it becomes complicated to really understand what is happening when. 
 
Example: https://github.com/grafana/grafana/blob/main/.github/commands.json#L35 this action does not exist at all, it only works because `addLabel` and `removeLabel` properties are set.

> * Does it support both org and repo projects?
Nope, new [projects](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects) only

**Comment by tolzhabayev** [comment]
@marefr changed the structure, for me it is good to go. waiting for your feedback.

**Review by marefr — APPROVED** [review]
LGTM 🚀

**Pull request merged**